src/toil/batchSystems/abstractGridEngineBatchSystem.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import logging
+import queue
```

Comment:
If we already import `Queue` and `Empty` later, we don't need the whole module probably.
```suggestion
```